src/app/_fetchers/AssigneeFetcher.tsx

```diff
@@ -37,5 +37,9 @@ const fetchAssignee = async (
 
 export const AssigneeFetcher = async ({ token, userType, viewSettings }: Props) => {
   const assignableUsersWithType = addTypeToAssignee(await fetchAssignee(token, userType))
-  return <ClientSideStateUpdate assignee={assignableUsersWithType} viewSettings={viewSettings} />
+  return (
+    <ClientSideStateUpdate assignee={assignableUsersWithType} viewSettings={viewSettings}>
+      {null}
+    </ClientSideStateUpdate>
```

Comment:
This doesn't hamper anything but it still looks weird :sweat_smile:

Reply:
Would you like me to revert this to what it was before? This was just a personal preference.